--- temp.out.norm	2026-05-14 17:34:10.642641281 +0300
+++ tests/out/test27.out.norm	2026-05-14 17:34:10.643377824 +0300
@@ -6,11 +6,11 @@
 OK
 OK
 OK
-BoydStevens EthanMatthews Victor
-BoydStevens Victor EthanMatthews
-BoydStevens Victor
+3 BoydStevens EthanMatthews Victor
+3 BoydStevens Victor EthanMatthews
+2 BoydStevens Victor
 OK
-BoydStevens EthanMatthews SaraMyers Victor
+4 BoydStevens EthanMatthews SaraMyers Victor
 colony 3 BoydStevens EthanMatthews Victor
 mystery 4 BoydStevens EthanMatthews SaraMyers Victor
 newcomer 2 EthanMatthews JimMatthews
