--- temp.out.norm	2026-05-14 17:34:10.661647414 +0300
+++ tests/out/test30.out.norm	2026-05-14 17:34:10.662391762 +0300
@@ -6,12 +6,12 @@
 OK
 OK
 OK
-Denji HayakawaAki MakimaSan PowerBlood
-MakimaSan Denji PowerBlood
-MakimaSan Denji
+4 Denji HayakawaAki MakimaSan PowerBlood
+3 MakimaSan Denji PowerBlood
+2 MakimaSan Denji
 OK
-Denji HayakawaAki MakimaSan
-MakimaSan Denji PowerBlood CosmosAngel
+3 Denji HayakawaAki MakimaSan
+4 MakimaSan Denji PowerBlood CosmosAngel
 angel 1 CosmosAngel
 blood 1 PowerBlood
 chainsaw 1 Denji
