--- temp.out.norm	2026-05-14 17:34:10.636164100 +0300
+++ tests/out/test26.out.norm	2026-05-14 17:34:10.636836415 +0300
@@ -7,11 +7,11 @@
 OK
 OK
 OK
-BiancaBarclay EnidSinclair WednesdayAddams XavierThorpe
-WednesdayAddams BiancaBarclay EnidSinclair
-WednesdayAddams XavierThorpe
+4 BiancaBarclay EnidSinclair WednesdayAddams XavierThorpe
+3 WednesdayAddams BiancaBarclay EnidSinclair
+2 WednesdayAddams XavierThorpe
 OK
-LurchAddams WednesdayAddams
+2 LurchAddams WednesdayAddams
 butler 1 LurchAddams
 detective 1 WednesdayAddams
 hyde 1 TylerGalpin
