--- temp.out.norm	2026-05-14 18:20:13.490287837 +0300
+++ tests/out/test40.out.norm	2026-05-14 18:20:13.491372283 +0300
@@ -17,3 +17,9 @@
 party 2 DustinHenderson MaxMayfield
 psychic 1 ElevenHopper
 upsidedown 1 ElevenHopper
+chief 1 HopperChief
+hawkins 4 DustinHenderson ElevenHopper HopperChief MaxMayfield
+hero 1 ElevenHopper
+party 2 DustinHenderson MaxMayfield
+psychic 1 ElevenHopper
+upsidedown 1 ElevenHopper
