--- temp.out.norm	2026-05-14 17:34:07.132643707 +0300
+++ tests/out/test55.out.norm	2026-05-14 17:34:07.133643713 +0300
@@ -36,12 +36,14 @@
 OK
 OK
 OK
+NOT FOUND
 afaceri 1 doc5
 brad 2 doc0 doc1
 munte 1 doc1
 palmieri 1 doc0
 prapastie 1 doc6
 OK
+NOT FOUND
 afaceri 1 doc5
 brad 2 doc0 doc1
 munte 1 doc1
@@ -56,6 +58,8 @@
 palmieri 1 doc0
 parcare 1 doc0
 prapastie 1 doc6
+NOT FOUND
+NOT FOUND
 OK
 afaceri 1 doc5
 ancora 1 doc0
@@ -65,11 +69,16 @@
 palmieri 1 doc0
 parcare 1 doc0
 prapastie 1 doc6
+NOT FOUND
 OK
+NOT FOUND
 OK
+NOT FOUND
 OK
+NOT FOUND
 OK
 OK
+NOT FOUND
 OK
 afaceri 1 doc5
 ancora 1 doc0
@@ -160,7 +169,9 @@
 violet 1 doc8
 OK
 OK
+NOT FOUND
 OK
+NOT FOUND
 aer 1 doc9
 afaceri 2 doc5 doc9
 ancora 2 doc0 doc12
@@ -183,19 +194,27 @@
 traseu 1 doc10
 umbrela 2 doc10 doc6
 violet 1 doc8
+NOT FOUND
 OK
 OK
+NOT FOUND
 OK
 OK
 OK
 OK
+NOT FOUND
 OK
+NOT FOUND
 OK
+NOT FOUND
+NOT FOUND
 OK
+NOT FOUND
 OK
 OK
 OK
 OK
+NOT FOUND
 OK
 OK
 OK
@@ -232,6 +251,7 @@
 umbrela 1 doc6
 violet 1 doc8
 OK
+NOT FOUND
 OK
 aer 1 doc9
 afaceri 1 doc9
