--- temp.out.norm	2026-05-14 17:34:10.648907602 +0300
+++ tests/out/test28.out.norm	2026-05-14 17:34:10.649634735 +0300
@@ -6,12 +6,12 @@
 OK
 OK
 OK
-ArminArlert LeviAckerman MikasaAckerman
-LeviAckerman MikasaAckerman ArminArlert
-ErenYeager
+3 ArminArlert LeviAckerman MikasaAckerman
+3 LeviAckerman MikasaAckerman ArminArlert
+1 ErenYeager
 OK
-ErenYeager ReinerBraun
-LeviAckerman MikasaAckerman
+2 ErenYeager ReinerBraun
+2 LeviAckerman MikasaAckerman
 ackerman 2 LeviAckerman MikasaAckerman
 eldian 2 ArminArlert ErenYeager
 founder 1 ErenYeager
