--- temp.out.norm	2026-05-14 17:54:37.362221073 +0300
+++ tests/out/test23.out.norm	2026-05-14 17:54:37.362898230 +0300
@@ -17,10 +17,3 @@
 5 ChloeFrager ElenaFisher NathanDrake SamDrake SullyMcaffrey
 2 NathanDrake SullyMcaffrey
 OK
-adventurer 5 ChloeFrager ElenaFisher NathanDrake SamDrake SullyMcaffrey
-hunter 1 NathanDrake
-journalist 1 ElenaFisher
-relic 1 NathanDrake
-smuggler 1 SullyMcaffrey
-thief 1 SamDrake
-treasure 2 NathanDrake SullyMcaffrey
