--- temp.out.norm	2026-05-14 18:09:03.301237938 +0300
+++ tests/out/test80.out.norm	2026-05-14 18:09:03.302237945 +0300
@@ -2,23 +2,23 @@
 OK
 OK
 2 d0 d1
-fluture 2 d0 d2
+2 d0 d2
 OK
 OK
 EMPTY
-orizont 3 d0 d2 d3
+3 d0 d2 d3
 EMPTY
 OK
 EMPTY
 3 d0 d2 d3
-busola 2 d0 d3
+2 d0 d3
 1 d2
 OK
 OK
 OK
-univers 2 d0 d4
+2 d0 d4
 1 d4
-azur 2 d2 d4
+2 d2 d4
 OK
 NOT FOUND
 OK
@@ -98,20 +98,10 @@
 zbor 1 d4
 2 d0 d4
 3 d0 d5 d4
-sarma 4 d0 d2 d4 d5
-scanteie 3 d0 d4 d5
-sclipici 3 d0 d4 d5
-scoica 2 d4 d5
-sfera 3 d0 d2 d4
-sidef 3 d0 d2 d5
-smarald 3 d0 d2 d5
-spectru 3 d2 d4 d5
-spirala 3 d0 d2 d4
-stea 1 d0
-stejar 3 d0 d2 d4
+4 d0 d2 d4 d5
 OK
 NOT FOUND
-jar 3 d2 d4 d6
+3 d2 d4 d6
 OK
 OK
 EMPTY
@@ -190,22 +180,21 @@
 vultur 1 d2
 zambet 1 d4
 zbor 1 d4
-zambet 1 d4
+1 d4
 EMPTY
 OK
 4 d2 d4 d6 d7
 EMPTY
 2 d2 d7
 OK
-cafea 2 d0 d2
-castel 2 d0 d5
+3 d0 d2 d5
 4 d0 d4 d5 d7
 OK
 OK
-elixir 4 d0 d5 d6 d8
+4 d0 d5 d6 d8
 1 d5
 OK
-sarma 4 d0 d2 d5 d8
+4 d0 d2 d5 d8
 OK
 3 d0 d5 d8
 EMPTY
@@ -213,21 +202,11 @@
 OK
 3 d0 d7 d8
 2 d0 d5
-sarma 4 d0 d2 d5 d8
-scanteie 4 d0 d5 d7 d8
-sclipici 3 d0 d5 d8
-scoica 1 d5
-sfera 4 d0 d2 d7 d8
-sidef 4 d0 d2 d5 d8
-smarald 4 d0 d2 d5 d7
-spectru 4 d2 d5 d7 d8
-spirala 3 d0 d2 d7
-stea 2 d0 d8
-stejar 2 d0 d2
+5 d0 d2 d5 d7 d8
 OK
 4 d0 d5 d7 d8
-smarald 4 d0 d2 d5 d7
-ceara 2 d0 d5
+4 d0 d2 d5 d7
+2 d0 d5
 OK
 OK
 OK
