--- temp.out.norm	2026-05-14 18:09:03.258325982 +0300
+++ tests/out/test74.out.norm	2026-05-14 18:09:03.259002028 +0300
@@ -1,67 +1,14 @@
 OK
 OK
-acceleratie 1 list1
-acustica 1 list1
-aerian 1 list2
-aerodinamica 1 list1
-alhazen 1 list1
-ambarcatiune 1 list2
-amestec 1 list1
-ampere 1 list1
-analizator 1 list1
-ancora 1 list2
-anihilare 2 list1 list2
-animal 1 list2
-animatie 1 list2
-anizotropie 1 list1
-anod 2 list1 list2
-anomalie 1 list2
-anorganic 1 list2
-ansamblu 1 list1
-antena 1 list1
-antineutron 1 list1
-antipa 1 list2
-aparat 2 list1 list2
-apogeu 1 list1
-archimede 1 list1
-aripi 1 list2
-aristotel 1 list1
-asachi 1 list2
-ascensiune 1 list2
-asincron 1 list1
-asociere 1 list1
-astrofizica 1 list1
-astronomie 1 list1
-atasament 1 list2
-atenuator 1 list2
-aterizare 1 list2
-atlas 1 list2
-atmosfera 1 list1
-atom 1 list1
-atractie 1 list1
-aurel 1 list2
-avioane 1 list2
-avogadro 1 list1
-analizator 1 list1
-aparat 2 list1 list2
-asachi 1 list2
-ascensiune 1 list2
-asincron 1 list1
-asociere 1 list1
-astrofizica 1 list1
-astronomie 1 list1
-astrofizica 1 list1
-astronomie 1 list1
+2 list1 list2
+1 list1
+2 list1 list2
+2 list1 list2
+1 list1
 OK
 EMPTY
 OK
-analizator 1 list1
-anihilare 1 list1
-anizotropie 1 list1
-anod 1 list1
-ansamblu 1 list1
-antena 1 list1
-antineutron 1 list1
+1 list1
 acceleratie 1 list1
 acustica 1 list1
 aerodinamica 1 list1
